### PR TITLE
Merge pull request #1116 from rapidsai/branch-0.15

### PR DESCRIPTION
**Pull request by BradReesWork**
[RELEASE] cugraph v0.15

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ Read the section on [building cuGraph from source](SOURCEBUILD.md) to validate t
 ```git remote add upstream https://github.com/rapidsai/cugraph.git```
 
 3. Checkout the latest branch
-cuGraph only allows contribution to the current branch and not main or a future branch.  Please check the [cuGraph](https://github.com/rapidsai/cugraph) page for the name of the current branch.  
+cuGraph only allows contribution to the current branch and not main or a future branch.  PLease check the [cuGraph](https://github.com/rapidsai/cugraph) page for the name of the current branch.
```
Comment (seunghwak) [inline CONTRIBUTING.md:98]: Why "PL"ease over "Pl"ease?

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

**Pull request closed without merging**